tests/component/test_read_properties.py

```diff
@@ -0,0 +1,163 @@
+"""Tests for validating Read properties."""
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx.constants import ReadRelativeTo
+from nidaqmx.task import Task
+
+
+@pytest.fixture(scope="module")
+def read_task(any_x_series_device):
+    """Gets AI voltage task."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+        yield task
+
+
+def test__write__get_int32_property__returns_value(read_task: Task):
```

Comment:
Please rename this test and following to `test__read...`

Reply:
Updated